test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,44 @@
+package seedu.addressbook.commands;
+
+import org.junit.Test;
+
+import seedu.addressbook.commands.Command;
+import seedu.addressbook.commands.SortCommand;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.Assert.*;
```

Comment:
Wildcard here as well

Reply:
Imports from java and other third parties should come before `seedu` imports

Reply:
Fixed.